.coveragerc

```diff
@@ -3,6 +3,7 @@ omit =
     tests/*
     examples/*
     setup.py
+    fiwareglancesync/scripts/*
```

Comment:
Wow, wow, wow.

If you want to do it omit fiwareglancesync/scripts/support but you have to leave getnids...